Source/AtomicJS/Javascript/JSAPI.cpp

```diff
@@ -473,6 +520,15 @@ void js_push_variant(duk_context *ctx, const Variant& v)
     case VAR_STRING:
         duk_push_string(ctx, v.GetString().CString());
         break;
+    case VAR_BUFFER:
+    {
```

Comment:
Excellent, I filed an issue to cleanup the scope usage in js_push_variant: https://github.com/AtomicGameEngine/AtomicGameEngine/issues/454